packages/react/toolbar/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@radix-ui/react-toolbar",
+  "version": "0.0.1",
+  "license": "MIT",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.module.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "scripts": {
+    "build": "parcel build src/index.ts --no-cache",
+    "clean": "rm -rf dist",
+    "version": "yarn version",
+    "prepublish": "yarn clean && yarn build"
```

We've changed these recently in all packages, so I think it should be this instead to be inline with the rest:

```suggestion
    "clean": "rm -rf dist",
    "prepublish": "yarn clean"
```